sql/moz-fx-data-shared-prod/monitoring_derived/jobs_by_organization_v1/query.py

```diff
@@ -48,8 +48,8 @@ def create_query(date, project):
           total_slot_ms ,
           transaction_id,
           user_email,
-          query_info.resource_warning ,
-          query_info.query_hashes.normalized_literals,
+          query_info.resource_warning as query_info_resource_warning,
+          query_info.query_hashes_normalized_literals as query_info_query_hashes_normalized_literals,
```

Comment:
It looks like this was changed after I ran my test query and r+'d the PR. This first identifier should probably be `query_info.query_hashes.normalized_literals `